src/content/docs/tutorial-customer-experience/establish-current-state.mdx

```diff
@@ -0,0 +1,168 @@
+
+* An insurance company that sells both car and house insurance.
+* A media company that has multiple streaming services or channels.
+* A travel company that provides car rental as well as hotel bookings.
+
+**Basic:** Break out performance by product by:
+* Faceting on `pageUrl`: Use this approach when you group multiple products into one browser app in New Relic.
+* Faceting by `appName`: Use this approach when you instrument each product as a separate web app.
+* Grouping by `appName` and then facet: Use this approach when you hsbr multiple apps in browser supporting one product.
+
+**Advanced:** Add product offerings as custom attributes to browser pages using [custom attributes](/docs/telemetry-data-platform/custom-data/custom-events/report-browser-monitoring-custom-events-attributes).
+    </Collapser>
+    <Collapser
+        id="Environment"
+        title="Environment"
+    >
+A naming convention that specifies the environment for browser apps during or after instrumentation . Well named browser apps specify products and/or functions as well as environment. Examples:
+* `account-management.prod`
+* `hotels-book.prod`
+* `car-insurance.uat`
+
+Use app naming conventions to note that the environment supports filtering data in both the UI and in dashboards. For more information, see our docs for [how to rename browser apps](/docs/browser/new-relic-browser/configuration/rename-browser-apps/).
+    </Collapser>
+    <Collapser
+        id="Team"
+        title="Team"
+    >
+In some organizations, a single team supports multiple products, while in others, a product is big enough to be supported by multiple teams. Report on team performance against customer experience or engagement by either adding the team name to the browser app name in New Relic (for example, `account-management.prod.unicorn-squad`) or by using [custom attributes](/docs/telemetry-data-platform/custom-data/custom-events/report-browser-monitoring-custom-events-attributes).
+    </Collapser>
+</CollapserGroup>        
+    </Step>
+    <Step>
+### Import the quality foundation dashboard
+
+<img
+  title="Customer experience quality foundation dashboard"
+  alt="Customer experience quality foundation dashboard"
+  src={cxQfDashboard}
+/>
+
+The next step is to create a dashboard that you'll use to measure and improve your customer experience against your baselines. To do this: 
+
+1. Go to the [Quality foundation quickstart](https://newrelic.com/instant-observability/customer-experience-quality-foundation/7a5739bf-30ee-4be9-9705-14871cafd7f4).
+2. Follow the instructions in the quickstart to download the dashboard.
+3. Customize it to fit how you plan to segment the data.
+
+<Callout variant="tip">
+Make sure to align the dashboard to lines of business or customer facing offerings rather than teams. This ensures optimization time is spent where it's most impactful. 
```

Comment:
"customer-facing"

The second sentence is too passive-voicey

Reply:
Good callouts, thank you!